.github/workflows/main.yml

```diff
@@ -67,4 +84,4 @@ jobs:
           api_key: ${{ secrets.DEPLOY_API_KEY }}
           project: "Ncron.Advanced"
           packages: |
-            Ncron.Advanced:${{ steps.build.outputs.octoversion_fullsemver }}
+            NCrontab.Advanced:${{ steps.build.outputs.octoversion_fullsemver }}
```

Comment:
This error is not captured originally when there is only 1 channel in Octopus Deploy project, so create release always succeed due to Octopus CLI auto resolve functionality.